src/main/kotlin/com/jfrog/conan/clionplugin/services/ConanPluginInit.kt

```diff
@@ -1,12 +1,12 @@
 package com.jfrog.conan.clionplugin.services
 
+import com.intellij.openapi.components.service
 import com.intellij.openapi.project.Project
 import com.intellij.openapi.startup.ProjectActivity
-import com.jfrog.conan.clionplugin.conan.ConanPluginUtils
 
 
 class ConanPluginInit : ProjectActivity {
     override suspend fun execute(project: Project) {
-        ConanPluginUtils.downloadCMakeProvider()
+        project.service<ConanService>().downloadCMakeProvider()
```

Comment:
If we are puttin the provider in the local folder maybe we should delay this until we add something to the project, like we do with the conanfile and conandata? WDYT?